### ISSUE TITLE
Long page titles are cropped in the Pages editor (UX enahncement)

### ISSUE THREAD

**Issue by OllyHodgson** [dnnsoftware/Dnn.AdminExperience#1007]
When working with nested pages, or pages with long names, the page name gets cropped after only 10 or 12 characters:

![image](https://user-images.githubusercontent.com/437224/40838570-b96aaeba-6596-11e8-9d94-e845f3084fd3.png)

There's no obvious way to see the full page name without clicking on and loading it in the the right pane.

Would it be better if the full page name could be shown when the user hovers over/taps/tabs to the link  (depending on their input device)?

**Comment by valadas** [comment]
I think if we just try to reduce some whitespace, reduce the indentation here and there and make it a tiny bit larger and add word wrapping, it could be usable. When you add all those few pixels, it can give easily one or two more words. Just eyeballing it, I can see probably 200px wasted, that is almost 2 words.

**Comment by OllyHodgson** [comment]
Agreed. IIRC the page titles are cropped before being rendered to the page (the text is actually `Ten Charac...`) so it'll be slightly more complicated than a CSS change.

**Comment by valadas** [comment]
@OllyHodgson you are unfortunately correct
